bunsen/bunsen-core/src/main/java/com/cerner/bunsen/definitions/StructureDefinitions.java

```diff
@@ -21,37 +22,37 @@
 public abstract class StructureDefinitions {
 
   // TODO make this FHIR version specific and hide it under an abstract method.
-  protected static final Set<String> PRIMITIVE_TYPES =
+  @VisibleForTesting
+  public static final Set<String> PRIMITIVE_TYPES =
       ImmutableSet.<String>builder()
           .add("id")
           .add("boolean")
           .add("code")
           .add("markdown")
           .add("date")
           .add("instant")
-          .add("datetime")
           .add("dateTime")
           .add("time")
-          .add("oid")
           .add("string")
+          .add("oid")
+          .add("xhtml")
           .add("decimal")
           .add("integer")
-          .add("xhtml")
           .add("unsignedInt")
           .add("positiveInt")
           .add("base64Binary")
           .add("uri")
           // TODO: Figure out why these are added to R4 resource definitions.
-          .add("http://hl7.org/fhirpath/System.Boolean")
           .add("http://hl7.org/fhirpath/System.String")
+          .add("http://hl7.org/fhirpath/System.Boolean")
           .add("http://hl7.org/fhirpath/System.Integer")
           .add("http://hl7.org/fhirpath/System.Long")
           .add("http://hl7.org/fhirpath/System.Decimal")
           .add("http://hl7.org/fhirpath/System.DateTime")
           .add("http://hl7.org/fhirpath/System.Time")
-          .add("http://hl7.org/fhirpath/System.Quantity")
```

Comment:
I am assuming the reason for removing the `Quantity` is that, it is not a primitive data type in R4

Reply:
Correct.